### ISSUE TITLE
running this site on Mac with Firefox - I can't log in. (ericb)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#179]
Issue reported by **ericb** at 2015-02-25 18:58:55 UTC
Time: 2015-02-25T18:53:44+00:00
Session ID: e1a981112f8a1f0d0dcc151228143419
Problem Page URL: [/manage_resources/new](/manage_resources/new)
Previous Page URL: [http://freereg2.freereg.org.uk/refinery/login](http://freereg2.freereg.org.uk/refinery/login)
Reported Issue:
My access works file using web browser Safari ( on mac) but  with Firefox, I cannot log on. - it gets stuck in a loop at userID + password if I chose NOT to remember my password .

It does work with Opera browser but their Bookmarks bar is next to impossible to set up and use! 

![Screenshot](http://freereg2.freereg.org.uk)

**Comment by Captainkirkdawson** [comment]
Default changed to miles

**Comment by PatReynolds** [comment]
I've tried this a few times and can't recreate it. Once the remember password pop up appears I click the x to close the box and just proceed as normal.

**Comment by Sherlock21** [comment]
[not replying via Github as that server won't respond at the moment.]

I tried this again, just now, and whilst I get the initial form, I can search, but I can't log on to Members

Tried Darren's workround,   but that did not unlock anything.

and I tried without clicking the X box and that did not progress it.
Meanwhile, i have alternative access from Opera. That's fine.
Suggest Close this now.

Ericb
